pom.xml

```diff
@@ -91,6 +91,12 @@
                 <artifactId>slf4j-api</artifactId>
                 <version>${org.slf4j.verison}</version>
             </dependency>
+            <!-- https://mvnrepository.com/artifact/com.google.api-client/google-api-client -->
+            <dependency>
+                <groupId>com.google.api-client</groupId>
+                <artifactId>google-api-client</artifactId>
+                <version>1.35.2</version>
```

Comment:
We have a way of defining versions using variables. Shall we follow that approach? 
Eg:
```
<version>${org.slf4j.verison}</version>
```

Reply:
Fixed 33432eaa57b944939b058bc9c82c79b3b3212163